pkg/prompts/upgraderiskreport/upgraderiskreport.go

```diff
@@ -26,37 +26,69 @@ import (
 )
 
 const gkeUpgradeRiskReportPromptTemplate = `
-You are a GKE expert, you have to upgrade a GKE cluster {{.clusterName}} in {{.clusterLocation}} location to {{.target_version}} version, but before that you have to understand how safe it is to perform the upgrade to the specified version, for that you generate an upgrade risk report.
+# GKE Upgrade Risk Report Generation
 
-You're providing a GKE Cluster Upgrade risk report for a specific GKE cluster, the report focuses on a specific GKE upgrade risks which may raise upgrading from the current cluster version to the specified target version.
+**1. Input Parameters:**
+  - Cluster Name: {{.clusterName}}
+  - Cluster Location: {{.clusterLocation}}
+  - Target Version: {{.targetVersion}}
 
-For fetching any in-cluster resources use kubectl tool and gcloud get-credentials.
+**2. Your Role:**
+You are a GKE expert. Your task is to generate a comprehensive upgrade risk report for the specified GKE cluster, analyzing the potential risks of upgrading from its current version to the 'Target Version'.
 
-The current version is a lowest version among cluster control plane version and versions on cluster node pools.
+**3. Primary Goal:**
+Produce a report outlining potential risks, and actionable recommendations to ensure a safe and smooth GKE upgrade. The report should be based on the changes introduced between the cluster's current control plane version and the 'Target Version'.
 
-You download GKE release notes (https://cloud.google.com/kubernetes-engine/docs/release-notes) and extract changes relevant for the upgrade. To download GKE release notes content, you use command line tool - lynx. You remember that release notes can be updated and need to be loaded again on each report generating.
+**4. Handling Missing Target Version:**
+If 'Target Version' is not provided:
+  a. State that the target version is required.
```

Comment:
Can you instruct the agent to recommend going with the latest available version? 

As we discussed offline, we need smart defaults and going with furthest available version is one of the options